javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,103 @@
+// FreeCodeCamp user: https://www.freecodecamp.org/catalina-tech
+
+
+
+// Age-ify (A future age calculator)
+
+let yearOfBirth;
+yearOfBirth = 1988;
+console.log(typeof yearOfBirth);
+
+let yearFuture;
+yearFuture = 2028;
+console.log(typeof yearFuture);
+
+let age = yearFuture - yearOfBirth;
+console.log("You will be" + " " + age + " " + "in" + " " + yearFuture); 
```

Comment:
You can combine the spaces with the rest of the string like this:
```suggestion
console.log("You will be " + age + " in " + yearFuture); 
```